src/bosh_openstack_cpi/lib/cloud/openstack/cloud.rb

```diff
@@ -149,11 +166,11 @@ def create_stemcell(image_path, cloud_properties)
     # @return [Hash] normalized properties
     def normalize_image_properties(properties)
       image_properties = {}
-      image_options = ['name', 'version', 'os_type', 'os_distro', 'architecture', 'auto_disk_config',
+      image_options = ['version', 'os_type', 'os_distro', 'architecture', 'auto_disk_config',
                        'hw_vif_model', 'hypervisor_type', 'vmware_adaptertype', 'vmware_disktype',
                        'vmware_linked_clone', 'vmware_ostype']
       image_options.reject { |image_option| properties[property_option_for_image_option(image_option)].nil? }.each do |image_option|
-        image_properties[image_option.to_sym] = properties[property_option_for_image_option(image_option)]
+        image_properties[image_option.to_sym] = properties[property_option_for_image_option(image_option)].to_s
```

Comment:
@ljfranklin Why do we need this? Did you face any issues without this conversion to String?